src/targets/gpu/device/fp_conversion.cpp

```diff
@@ -0,0 +1,27 @@
+#include <migraphx/gpu/device/fp_conversion.hpp>
+#include <migraphx/gpu/device/nary.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace gpu {
+namespace device {
+
+argument
+fp_conversion(hipStream_t stream, const shape& output_shape, const std::vector<argument>& args)
```

Comment:
This should be `fp_conversion(hipStream_t stream, const argument& result, const argument& arg)` not a vector of arguments.